installer_scripts/install-yb-voyager

```diff
@@ -850,6 +854,25 @@ create_gather_assessment_metadata_dir() {
 	fi
 }
 
+create_guardrail_scripts_dir() {
+	scripts_parent_dir="/opt/yb-voyager"
+	scripts_dir_name="guardrails-scripts"
+	sudo mkdir -p $scripts_parent_dir/$scripts_dir_name
+
+	output "Installing the guardrails scripts"
+	if [ "${VERSION}" == "latest" ]
+	then
+		TAR_URL="https://github.com/yugabyte/yb-voyager/raw/$YB_VOYAGER_GIT_HASH/${scripts_dir_name}.tar.gz"
```

Comment:
You need to do it for the individual file for now as there is no tar zip in the repo for `gurardrails-scripts` folder.